Source/devices/oni/AcqBoardONI.cpp

```diff
@@ -1108,7 +1114,10 @@ float AcqBoardONI::getBitVolts (ContinuousChannel::Type channelType) const
         case ContinuousChannel::AUX:
             return 0.0000374;
         case ContinuousChannel::ADC:
-            return 0.00015258789;
+            if (deviceId == DEVICE_ID_V2)
```

Comment:
Make this an if dev == v3 return X else return Y
If by some chance the id was invalid or different, this will method will return an undefined value (I'm even surprised the compiler let you get away)